tests/integration/tests.mk

```diff
@@ -126,5 +126,5 @@ test.integration.kube.environment: | $(JUNIT_REPORT) check-go-tag
 ifeq (${JOB_TYPE},postsubmit)
 	$(call run-test,./tests/integration/...)
 else
-	$(call run-test,./tests/integration/security/ ./tests/integration/pilot,-run="TestReachability|TestTraffic|TestGatewayConformance")
+	$(call run-test,./tests/integration/...)
```

Comment:
I think this was temporary?

Reply:
I reverted now the full-scope testing as all the test have passed.
The one test that failed did so because I forgot to update the lib.sh script to build docker.ztunnel in presubmit